l10n/poeditor-upload/update-all-languages.py

```diff
@@ -139,6 +141,10 @@ def identify_terms_to_update(local_langs_data, poeditor_langs_data, langs=['en']
     locally_translated_terms = {lang: local_terms_with_translation[lang] & poeditor_terms_lacking_translation[lang] for lang in langs}
 
     terms_to_update = {lang: new_terms_with_translation[lang] | locally_translated_terms[lang] for lang in langs}
+
+    if verbose:
+        print('The following terms on POEditor do not exist locally:')
+        for term in sorted(list(poeditor_orphan_terms)): print('\t{}'.format(term))
```

Comment:
Why is this loop just one line? I would prefer it be written normally, on two lines.